exportdb/permissions.py

```diff
@@ -1,10 +1,11 @@
 import rules
-# TODO: this should be parametrized via settings
-
+from django.conf import settings
 
 @rules.predicate
-def is_superuser(user):
-    return user.is_superuser
+def permission(*args, **kwargs):
+    predicate = getattr(settings, 'EXPORTDB_PERMISSION', rules.is_superuser)
```

Comment:
I started out with something like this (without the appconf stuff, totally missed that). But we need `rules.is_group_member('something')` as a rule. Which will not work with `import_string`, that's why I choose this option.